src/_shared/form/components/Input/Input.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FC } from "react";
+import { useController, Controller } from "react-hook-form";
+import { Typography, TextField } from "../../../components";
+import { Stack } from "../../../components";
+import { useTranslate } from "../../../i18n";
+
+interface Props {
+  name: string;
+  label: string;
+}
+
+const Input: FC<Props> = ({ name, label }) => {
+  const translate = useTranslate();
+
+  const {
+    fieldState: { error, invalid },
+  } = useController({
+    name,
+  });
+
+  return (
+    <Stack direction="column">
+      <Controller
+        name={name}
+        render={({ field: { onChange, onBlur, value } }) => (
```

Comment:
is Controller required?

Reply:
React-hook-form docs suggest usign it with external library components. I think you can avoid using it with the inputProps of TextField but I think it will be a little bit tricker

Reply:
but we are using useController in the same component to get the fields, info: let's check it

Reply:
Ohh yeah, using them both is redundant! I've removed useControlled and used only the Controller component!